src/core/server/saved_objects/import/validate_references.ts

```diff
@@ -73,7 +73,10 @@ export async function getNonExistingReferenceAsKeys(
   }
 
   // Fetch references to see if they exist
-  const bulkGetOpts = Array.from(collector.values()).map((obj) => ({ ...obj, fields: ['id'] }));
+  const bulkGetOpts = Array.from(collector.values()).map((obj) => ({
```

Comment:
If there is no change for this line, can we keep the original, as it is likely your VS code formatting style.